frontend/src/components/App/App.tsx

```diff
@@ -84,6 +85,9 @@ const App = () => {
                         {/* Experiment */}
                         <Route path={URLS.experiment} element={<Experiment />} />
 
+                        {/* ExperimentAbout */}
+                        <Route path={URLS.experimentAbout} element={<ExperimentAbout />} />
+
```

Comment:
I am a bit confused :-) Is this supposed to be an about page for an experiment? In that case, I don't think this route will work properly as there the route / page component won't know for which experiment they will have to show the about page. I think it might be better to modify the `Experiment` component's logic to decide whether we want to show the consent already if the user wants to go to the experiment's about page.

Either here by checking the current location and see if the current location contains `"/about"`:

https://github.com/Amsterdam-Music-Lab/MUSCLE/blob/b15fd1032a63e746f8ecb818d9ba15ff2b2a8aca/frontend/src/components/Experiment/Experiment.tsx#L59-L71

Or here, by moving the logic that decides whether we want to show the consent form to a separate "Guard Component" (and something similar for the logic that decides whether participant will be redirected to next experiment instead of the dashboard):

```tsx
export const RequireConsent = ({ experiment, hasShownConsent, children }: RequireConsentProps) => {
    const showConsent = experiment?.consent;
    
    if (!hasShownConsent && showConsent) {
        return <Navigate to={`/experiment/${experiment.slug}`} replace />;
    }

    return <>{children}</>;
};
```

```tsx
export const RequireDashboardOrRedirect = ({ experiment, children }: RequireDashboardOrRedirectProps) => {
    const displayDashboard = experiment?.dashboard.length;
    const nextBlock = experiment?.nextBlock;
    const participantIdUrl = experiment?.participantIdUrl;
    
    if (!displayDashboard && nextBlock) {
        return <Navigate 
            to={`/block/${nextBlock.slug}${participantIdUrl ? `?participant_id=${participantIdUrl}` : ""}`} 
            replace 
        />;
    }

    return <>{children}</>;
};
```

```tsx
            <Routes>
                {/* About route is always accessible */}
                <Route
                    path="/about"
                    element={<ExperimentAbout content={experiment?.aboutContent} slug={experiment.slug} />}
                />
                
                {/* Main experiment route with nested guards */}
                <Route
                    path="*"
                    element={
                        <RequireConsent experiment={experiment} hasShownConsent={hasShownConsent}>
                            <RequireDashboardOrRedirect experiment={experiment}>
                                <ExperimentDashboard 
                                    experiment={experiment} 
                                    participantIdUrl={participantIdUrl} 
                                    totalScore={experiment.totalScore} 
                                />
                            </RequireDashboardOrRedirect>
                        </RequireConsent>
                    }
                />
            </Routes>
```